packages/angular/projects/ui-angular/src/lib/services/__tests__/authenticator.service.test.ts

```diff
@@ -47,7 +47,8 @@ describe('AuthenticatorService', () => {
 
   it('subscribe returns state machine facade', () => {
     const handler = jest.fn();
-    const { unsubscribe } = authService.subscribe(handler);
```

Comment:
Did some research here. So technically, destructuring here is not recommended because `subscribe(handler)` returns a class and destructured class function does not preserve `this`.

But this code actually runs fine in practice, because `xstate` and `rxjs` use some techniques so that `this` doesn't get lost:

- Xstate uses arrow functions to preserve `this`: [L464-L469](https://github.com/statelyai/xstate/blob/de75010fd943900eb1c776a87c2283531e46cbf0/packages/core/src/interpreter.ts#L464-L469)
- rxjs uses `function.call(this)` to preserve `this`: [L92](https://github.com/ReactiveX/rxjs/blob/adc351e3f7078300cafbe37a695b30c9bb1ae872/src/internal/Subscription.ts#L92)

Reply:
With that said though, updated code to make `eslint` happy 😛